clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1304,6 +1329,20 @@ template <typename T> class trace_converter_t {
         }
         impl()->log(4, "Appended memref type %d size %d to " PFX "\n", buf->type,
                     buf->size, (ptr_uint_t)buf->addr);
+
+#ifdef AARCH64
+        // TODO i#4400: Following is a workaround to correctly represent DC ZVA in
+        // offline traces. This doesn't help with online traces, and also causes us
+        // to lose the actually written address. Ideally, we want the actually
+        // written address and also the correct size and type in the IR.
+        if (instr->is_aarch64_dc_zva()) {
+            buf->addr =
```

Comment:
Best to handle a trace that doesn't have the cache line marker (not just some corner case: compatibility mismatches happen on any update due to trace gathering using a different build from what postprocessing uses): will this turn the address into 0 with size 0 as-is?  Should probably have the header code above just use the current host cache line size as an estimate.

Reply:
The header processing code in raw2trace.cpp now sets the current processor cache line size as default if the header doesn't contain it.